crates/shared/src/oneinch_api.rs

```diff
@@ -553,15 +553,14 @@ async fn logged_query<D>(client: &Client, url: Url) -> Result<D, OneInchError>
 where
     D: DeserializeOwned,
 {
-    tracing::trace!("Query 1inch API for url {}", url);
+    tracing::trace!(%url, "Query 1inch API");
     let response = client.get(url).send().await?.text().await?;
+    tracing::trace!(%response, "Received 1Inch API response");
+
```

Comment:
I think it makes slightly more sense to always log it in case of an error instead of having to opt-in into the `trace` logs. Then you can not lose any unexpected responses.
But this is fine as well.

Reply:
Personally, I think it is too noisy. We can always enable `trace` logs by default in our configuration if we want this behaviour.

Reply:
Oh, I didn't mean "always log every response" (I agree, too noisy) but rather "always log every response we weren't able to deserialize".